src/CoreFoundation/CFException.cs

```diff
@@ -90,7 +93,7 @@ static CFExceptionDataKey ()
 #endif
 	public class CFException : Exception {
 
-		public CFException (string description, NSString domain, nint code, string failureReason, string recoverySuggestion)
+		public CFException (string? description, NSString? domain, nint? code, string? failureReason, string? recoverySuggestion)
```

Comment:
You can't make the `code` argument nullable, because it's a struct and it would be a breaking change.